.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.8']
+        toxenv: ['core', 'needle', 'doc']
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
This should probably be `v1` to avoid problems with codecov, see https://openedx.atlassian.net/wiki/spaces/SRE/pages/2038792440/How+to+migrate+tests+from+TravisCI+to+Github+Actions for details.